src/app/[propertyId]/tour/components/timeline.tsx

```diff
@@ -0,0 +1,81 @@
+"use client";
+
+import Image from "next/image";
+
+import PlayIcon from "@/assets/icons/play.svg";
+import FallbackImage from "@/assets/img/location-image-fallback.png";
+import { TimeSlots } from "@/enums/app.enum";
+import useMapStore from "@/stores/useMapStore";
+import { useTourStore } from "@/stores/useTourStore";
+import { timeSlotConfigMap } from "@/utils/tour.util";
+
+const { setCurrentLocationData } = useMapStore.getState();
+
+export default function Timeline() {
+  const selectedItinerary = useTourStore((state) => state.selectedItinerary);
+  const mapInstance = useMapStore((state) => state.mapInstance);
+
+  return (
+    <div className="absolute -top-2 left-1/3 flex items-center justify-between gap-10">
+      {Object.entries(selectedItinerary).map(([timeSlot, activity]) => {
+        const timeSlotImage = timeSlotConfigMap[timeSlot as TimeSlots].icon;
+
+        if (!activity) {
+          return null;
+        }
+
+        const { place, generatedImage } = activity;
+
+        const { coordinates, name, distance, rating } = place;
+
+        const { x: lat, y: lng } = coordinates;
+
+        const replayItem = () => {
+          mapInstance?.setConfigProperty(
+            "basemap",
+            "lightPreset",
+            timeSlotConfigMap[timeSlot as TimeSlots].lightPreset
+          );
+
+          setCurrentLocationData({
+            coordinates: [lng, lat],
+            name,
+            distance,
+            rating,
+            image: generatedImage,
+          });
+
+          if (mapInstance) {
+            mapInstance.flyTo({
+              center: [lng, lat],
+              zoom: 18,
+              pitch: 60,
+              bearing: 30,
+              duration: 5000,
+            });
+
+            setTimeout(() => {
+              mapInstance.easeTo({
+                center: [lng, lat],
+                zoom: 18,
+                pitch: 45,
+                bearing: -30,
+                duration: 3000,
+              });
+            }, 5000);
+          }
+        };
+
+        const imageToShow = generatedImage ? `data:image/jpeg;base64,${generatedImage}` : FallbackImage;
```

Comment:
add src in fallback images also,
```suggestion
        const imageToShow = generatedImage ? `data:image/jpeg;base64,${generatedImage}` : FallbackImage.src;
```